labels/selector.go

```diff
@@ -87,14 +94,15 @@ func NewMustRegexpMatcher(name, pattern string) Matcher {
 
 }
 
-// notMatcher inverts the matching result for a matcher.
-type notMatcher struct {
+// NotMatcher inverts the matching result for a matcher.
+type NotMatcher struct {
```

Comment:
Please do not change the public API, @brian-brazil  ((:

Reply:
It does not cost us to just leave it public and most likely Thanos & Cortex might using it. (: